interfaces/cython/cantera/examples/reactors/ic_engine.py

```diff
@@ -125,15 +125,15 @@ def piston_speed(t):
     # define opening and closing of valves and injector
     if (np.mod(crank_angle(t_i) - inlet_open, 4 * np.pi) <
             np.mod(inlet_close - inlet_open, 4 * np.pi)):
-        inlet_valve.set_valve_coeff(inlet_valve_coeff)
+        inlet_valve.valve_coeff = inlet_valve_coeff
         test[n1] = 1
     else:
-        inlet_valve.set_valve_coeff(0)
+        inlet_valve.valve_coeff = 0.
```

Comment:
Now that the valve opening can be specified as a function of time, can't we define this as a function, and resolve #665?

Reply:
Exactly! That was the incentive for this PR.